frontend/src/components/dashboard/varianceHomePage/VarianceSearch.js

```diff
@@ -72,6 +64,7 @@ export class VarianceSearch extends Component {
                 initialValues={this.props.initialValues}
                 complianceCodes={this.props.complianceCodes}
                 mineRegionOptions={this.props.mineRegionOptions}
+                enableReinitialize
               />
```

Comment:
Same here, move this into the default export of the form

Reply:
done